package_linter.py

```diff
@@ -438,6 +447,46 @@ def license_mentionned_in_readme(path):
                     "already be playing with are only available on 3.x..."
                 )
 
+    def check_issues(self):
+
+        manifest = os.path.join(self.path, 'manifest.json')
+        with open(manifest, encoding='utf-8') as data_file:
+            manifest = json.loads(data_file.read(), object_pairs_hook=check_for_duplicate_keys)
+
+        app_id = manifest["id"]
+        catalog = requests.get("https://raw.githubusercontent.com/YunoHost/apps/master/apps.json").json()
+        if app_id not in catalog:
+            print_error("Your app does not appears to be listed in YunoHost's app catalog.")
+            return
+        elif catalog[app_id]["state"] != "working":
+            print_warning("Your app is not flagged as 'working' in Yunohost's app catalog.")
+
+        repo_url = catalog[app_id]["url"]
+        if "github.com" not in repo_url:
+            print_warning("Can't check if there are any blocking issues pending, can only do this for apps hosted on github for now.")
+            return
+
+        uri = 'repos/{}/issues'.format(repo_url.replace("https://github.com/", ""))
+
+        r = requests.get('https://api.github.com/{}'.format(uri)).json()
+        if "message" in r and r["message"] == "Not Found":
+            print('https://api.github.com/{}'.format(uri))
+            print(r["message"])
+            return
+
+        issues = [ i["title"] for i in r if not "pull_request" in i ]
+
+        blocking_issues = [ i for i in issues if i.upper().startswith("[LOW QUALITY]") or i.upper().startswith("[BROKEN]") or i.startswith("E") ]
```

Comment:
What do you mean with `or i.startswith("E")` ? I think it's a bit of a too broad test.

Reply:
Ugh yes sorry that was just me adding this to validate the code was working using a random app that does not have any issue like this yet :sweat_smile: Sorry about that, when writing this I was like "Ugh I must absolutely not forget to remove this when commiting" xD